docs/edge.rst

```diff
@@ -318,19 +318,177 @@ the form.
 
 .. image:: _assets/img/cloud-edge-delete.png
    :alt: CrateDB Edge deletion confirmation screen
+    :scale: 20%
 
 Deleting a custom region does not delete the resources inside that region. To
 also delete the resources inside the region, run the script provided in the
 deletion confirmation screen in your local CLI. This will uninstall CrateDB
 Cloud Edge from your local Kubernetes cluster.
 
 
+.. _edge-tools:
+
+Install CrateDB Edge with third-party tools
+===========================================
+
+In the tutorial walkthroughs below, you can read how to install CrateDB Edge
+using two of the most common Kubernetes distributions: `Microk8s`_ and `K3s`_.
+These are third-party tools and not officially supported by Crate.io, nor are
+we responsible for their behavior. That said, we have tested the instructions
+provided below for functionality. Users less familiar with customizing their
+Kubernetes stack on their own may find either of these two guides a practical
+solution for easier CrateDB Edge setup.
+
+
+.. _edge-tools-microk8s:
+
+Microk8s
+--------
+
+Below is a full walkthrough of the steps required to get CrateDB Edge up and
+running on Microk8s. The steps are merely examples of a process validated by
+us, but we do not guarantee its use, nor is it the only possible method for
+CrateDB Edge on Microk8s. As mentioned, we do not officially support Microk8s
+or other third-party tools. We provide this information 'as is', for ease of
+use and to illustrate the workings of CrateDB Edge.
+
+
+Set up Microk8s
+'''''''''''''''
+
+Follow the instructions from the `Microk8s docs`_:
```

Comment:
I would also add that "In this tutorial, we'll be setting up a three node Kubernetes cluster. You can also use a single node for testing purposes."